check-jenkins-build-time/lib/check_jenkins_build_time.go

```diff
@@ -0,0 +1,137 @@
+package checkjenkinsbuildtime
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/mackerelio/checkers"
+)
+
+var opts struct {
+	Scheme        string `short:"s" long:"scheme" default:"http" description:"Jenkins scheme"`
+	Host          string `short:"h" long:"host" default:"localhost" description:"Jenkins hostname"`
+	Port          int64  `short:"p" long:"port" default:"8080" description:"Jenkins port"`
```

Comment:
I think it's more convenient to have Jenkins base url configuration instead of specifying these three options.